Sources/SCNPath/RKPathEntity.swift

```diff
@@ -0,0 +1,215 @@
+//
+//  RKPathEntity.swift
+//  SCNPath+Example
+//
+//  Created by Grant Jarvis on 3/13/21.
+//  Copyright © 2021 Max Cobb. All rights reserved.
+//
+
+import SceneKit
+import Combine
+import RealityKit
+
+public class RealityKitPathEntity : SCNPathNode {
```

Comment:
I think the name RKPathEntity is good, could you change the class name here?

Reply:
Done!